### ISSUE TITLE
Pages' content and styling

### ISSUE THREAD

**Issue by dannycolin** [OpenScienceMOOC/site#37]
A few pages on the new website need content.

**Tasks** 
- [x] All: Increase line-height and paragraph spacing
- [x] All: Increase color contrast (green on black)
- [x] People: Update list
- ~~[x] Get-Involved: Decrease headings font size~~
- [x] Hero: Slogan on two lines instead of three
- ~~[x] Hero: Change "enroll" for something catchier~~
- [x] Footer: Increase top spacing
- ~~[x] Mission: Rephrase the "about" text to look more like an organization mission statement.~~
- [x] Modules: Add a ToC sidebar

**Comment by Protohedgehog** [comment]
Awesome, thanks for this @dannycolin. Yes, having a mission statement would be better. What's the best way to contribute this content atm for you?

**Comment by dannycolin** [comment]
@Protohedgehog you can create a PR and compare it with the v1.0 branch.

**Comment by Protohedgehog** [comment]
@dannycolin Just realised I closed this tab by mistake without doing the work. Will get on it ASAP.

**Comment by Protohedgehog** [comment]
@dannycolin Just realised I closed this tab by mistake without doing the work. Will get on it ASAP.

**Comment by Protohedgehog** [comment]
Need a catchy sentence or two (homepage)
_The Open Science MOOC is about transforming the way we do research based on essential values, principles, and practices. Our mission is to set the default setting for global research to Open._

    Spread the Word: Open Science is for everyone. Help us spread the word by joining our [open Slack channel](https://osmooc.herokuapp.com) and inviting your friends and colleagues to be part of our community.
    Join our Team: Everything we develop and produce is in the open. You can join our growing team on [GitHub](https://open-science-mooc-invite.herokuapp.com/). If you want to become an instructor, [email us](mailto:info@opensciencemooc.eu)!
    Become a partner: We have three different partner levels: Core partners to help with the critical development of the MOOC; strategic partners who we work with to advance Open Science together; and institutional partners, who can help locally deliver the MOOC modules.

A more detailed text on how to join the MOOC (production team, steering committee, partners, sponsors, etc.):

How to get involved! There are a number of ways to engage with the MOOC:
Steering Committee - To help focus the strategic direction of the whole project.
Executive Institutional Committee: Guides the co-ordination and implementation of the MOOC at the institute level. Contact info@opensciencemooc.eu if you are interested here.
Production team:  Each module has a core development team. If you would like to join, there are two ways. Firstly, simply by adding yourself to the open GitHub teams. And secondly, contacting us directly to be part of the core development team for each module.
Strategic partners: Groups or organisations who we can work with together to help advance Open Science together. If you believe that your work is aligned with that of the MOOC, come and join us in making Open Science a reality!
Sponsors: The Open Science MOOC is a volunteer-driven project. We rely on the goodwill and time of a huge number of people working together to work. We have received a grant from CodeOcean to help with the technical aspects of the MOOC (e.g., purchasing equipment), and a Shuttleworth flash grant to our founder to help boost the project. We keep costs at an absolute minimum, but if you believe in what we do and want to help out, we appreciate any financial assistance!  

@dannycolin is this okay for now? Can do the mission statement later. Sun is out now and I want to play in the garden.. ;)

**Comment by dannycolin** [comment]
Yep. I've only rephrase "become a partner" and move the partner level definitions to the "get involved" page because the block was twice the size as the others. 

There's a few things here and there that I need to fix and we'll be ready to launch the website in prod. Should we share the temporary link to the steering committee just to have some feedbacks?

Oh and I hope you had some fun today in your garden! :D Here, there's still a bit a snow covering the grass :(.

**Comment by Protohedgehog** [comment]
OK, sweet. So I can make edits to this branch https://github.com/OpenScienceMOOC/site/tree/v1.0 and make a PR for changes, right? Will familiarise myself with the new site structure if so.

And yes, I think sharing would be a good idea! I'll DM you the SC mailing list :)

**Comment by dannycolin** [comment]
> OK, sweet. So I can make edits to this branch https://github.com/OpenScienceMOOC/site/tree/v1.0 and make a PR for changes, right? Will familiarise myself with the new site structure if so.

Yes. If you have questions about the structure don't hesitate to DM me on Slack ;).

**Comment by johav** [comment]
Just c&p Lisa's comments:

Homepage
- Would it be possible to have "We want to make open the default setting for all global research" on two lines instead of three? 
- Can we think of something catchier than "enroll"? (nice color change on hover btw!)
- The list of modules under "Discover our courses" (I like the background!): consistent margins & capitalization 
- Footer: Is it possible to shift the content further down?
- I really like the little icons, and the color on hover again :)

Get Involved
- Consistent capitalization & British or American English, "Partnerships" is missing the "s"
- Increase the spacing between each section
- To make it easier to read and seem less overwhelming, I would have the h3s sized like h4s?

Our Mission
- Could you increase the spacing between the paragraphs?

**Comment by johav** [comment]
The website looks quite awesome! Thanks a lot :)
- where will the OSMOOC logo be? first setion on home page?
<img width="547" alt="image" src="https://user-images.githubusercontent.com/13553996/55536406-f623f180-56b9-11e9-90a4-4dc1ff46c7ac.png">

- can we keep the list of resources on the website?
- Our Partners: add partner orgs logos (need to distinguish between moral and financial support partners?)
- update team & steering committee

**Comment by tosteiner** [comment]
@dannycolin chiming in with the others: amazing stuff re website, this looks really great!

One thing I stumbled upon: should we maybe also add GDPR-related links to the footer? Sorry, I know this is PITA territory ...

![](https://media.giphy.com/media/l4WjpCogXmWxq/giphy.gif)

Not sure if e.g. adapting the `privacy policy` part from FOSTER https://www.fosteropenscience.eu/privacy-policy would help?

**Comment by dannycolin** [comment]
> where will the OSMOOC logo be? first setion on home page?

The logo (top right part of the image) will replace "Open Science MOOC" in the topbar menu. 

> can we keep the list of resources on the website?

Each module has a resources section based on the resources page. I thought it would be easier to digest the information but we could recreate a resources page if needed. 

> Our Partners: add partner orgs logos (need to distinguish between moral and financial support partners?)

@Protohedgehog any idea on that one. In your text, you've divided partners status in three types?

> One thing I stumbled upon: should we maybe also add GDPR-related links to the footer?

Oh yes. I think making sure the website is GDPR-compliant is a good idea. I don't know a lot about it tho. I've opened an issue (#40) specifically for that.

**Comment by Protohedgehog** [comment]
The modular breakdown of the resources section is pretty good. Is there a way of making each one look a bit better for navigation? Maybe just adding a simple ToC?

Can we keep the partner logos divided up in just the same way as they are on the old site for now too?

**Comment by dannycolin** [comment]
> The modular breakdown of the resources section is pretty good. Is there a way of making each one look a bit better for navigation? Maybe just adding a simple ToC?

Yep. I just hit a weird bug with Jekyll ToC but I'll add it to the list of tasks.

**Comment by dannycolin** [comment]
All done. :)

**Comment by Protohedgehog** [comment]
Legend, thanks @dannycolin! :)
